build.sbt

```diff
@@ -136,13 +137,119 @@ lazy val apiContractJS =
+  antlr.jvm
+    .in(file("./amf-antlr-syntax/jvm"))
+    .sourceDependency(antlrv4JVMRef, antlrv4LibJVM)
+lazy val antlrJS =
+  antlr.js
+    .in(file("./amf-antlr-syntax/js"))
+    .disablePlugins(SonarPlugin, ScalaJsTypingsPlugin)
+    .sourceDependency(antlrv4JSRef, antlrv4LibJS)
+
+/** **********************************************
+  * AMF-GRPC
+  * ********************************************* */
+
+lazy val grpc = crossProject(JSPlatform, JVMPlatform)
+  .settings(
+    Seq(
+      name := "amf-grpc"
+    ))
+  .in(file("./amf-grpc"))
+  .settings(commonSettings)
+  .dependsOn(apiContract, antlr)
+  .jvmSettings(
+    libraryDependencies += "org.scala-js"                      %% "scalajs-stubs"         % scalaJSVersion % "provided",
+    artifactPath in (Compile, packageDoc) := baseDirectory.value / "target" / "artifact" / "amf-grpc-javadoc.jar",
+    mappings in (Compile, packageBin) += file("amf-apicontract.versions") -> "amf-apicontract.versions"
+  )
+  .jsSettings(
+    scalaJSModuleKind := ModuleKind.CommonJSModule,
+    artifactPath in (Compile, fullOptJS) := baseDirectory.value / "target" / "artifact" / "amf-grpc.js",
+    scalacOptions += "-P:scalajs:suppressExportDeprecations"
+  )
+  .disablePlugins(SonarPlugin)
+
+lazy val grpcJVM =
+  grpc.jvm
+    .in(file("./amf-grpc/jvm"))
+lazy val grpcJS =
+  grpc.js
+    .in(file("./amf-grpc/js"))
+    .disablePlugins(SonarPlugin, ScalaJsTypingsPlugin)
+
+/** **********************************************
+  * AMF-GRAPHQL
+  * ********************************************* */
+
+lazy val graphql = crossProject(JSPlatform, JVMPlatform)
+  .settings(
+    Seq(
+      name := "amf-graphql"
+    ))
+  .in(file("./amf-graphql"))
+  .settings(commonSettings)
+  .dependsOn(apiContract, antlr)
+  .jvmSettings(
+    libraryDependencies += "org.scala-js"                      %% "scalajs-stubs"         % scalaJSVersion % "provided",
+    artifactPath in (Compile, packageDoc) := baseDirectory.value / "target" / "artifact" / "amf-graphql-javadoc.jar",
+    mappings in (Compile, packageBin) += file("amf-apicontract.versions") -> "amf-apicontract.versions"
+  )
+  .jsSettings(
+    scalaJSModuleKind := ModuleKind.CommonJSModule,
+    artifactPath in (Compile, fullOptJS) := baseDirectory.value / "target" / "artifact" / "amf-graphql.js",
+    scalacOptions += "-P:scalajs:suppressExportDeprecations"
+  )
+  .disablePlugins(SonarPlugin)
+
+lazy val graphqlJVM =
+  graphql.jvm
+    .in(file("./amf-graphql/jvm"))
+lazy val graphqlJS =
+  graphql.js
+    .in(file("./amf-graphql/js"))
+    .disablePlugins(SonarPlugin, ScalaJsTypingsPlugin)
+
 /** **********************************************
   * AMF CLI
   * ********************************************* */
 lazy val cli = crossProject(JSPlatform, JVMPlatform)
   .settings(name := "amf-cli")
   .settings(fullRunTask(defaultProfilesGenerationTask, Compile, "amf.tasks.validations.ValidationProfileExporter"))
-  .dependsOn(apiContract)
+  .dependsOn(grpc)
```

Comment:
Doesn't anyone depend on graphql?

Reply:
CLI will depend on graphql, but I didn't finish the minimun support yet